.editorconfig

```diff
@@ -130,19 +151,32 @@ csharp_space_between_method_declaration_parameter_list_parentheses = false
 csharp_space_between_parentheses = false
 csharp_space_between_square_brackets = false
 
-[*.{asm,inc}]
-indent_size = 8
+# Analyzers
+dotnet_code_quality.ca1802.api_surface = private, internal
+
+# C++ Files
+[*.{cpp,h,in}]
+curly_bracket_next_line = true
+indent_brace_style = Allman
 
 # Xml project files
 [*.{csproj,vcxproj,vcxproj.filters,proj,nativeproj,locproj}]
 indent_size = 2
 
-# Xml config files
-[*.{props,targets,config,nuspec}]
+# Xml build files
+[*.builds]
 indent_size = 2
 
-[CMakeLists.txt]
+# Xml files
+[*.{xml,stylecop,resx,ruleset}]
 indent_size = 2
 
-[*.cmd]
+# Xml config files
+[*.{props,targets,config,nuspec}]
 indent_size = 2
+
+# Shell scripts
+[*.sh]
+end_of_line = lf
```

Comment:
📝 This is primarily controlled by **.gitattributes**. The value seen here would really only apply to certain local-development edge cases related to the creation of new files.